src/TypeScript.fs

```diff
@@ -2642,7 +2642,9 @@ module Ts =
     type [<AllowNullLiteral>] Statement =
         inherit Node
         abstract _statementBrand: obj option with get, set
-
+        //added for ts2fable
+        abstract nextContainer: Statement option
+        abstract symbol: Statement option
```

Comment:
Is there a way to do this without modifying the generated files?

Reply:
Remove nextContainer & symbol
This is useless